spec/models/wallet_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Wallet, type: :model do
+  describe "Associations" do
+    it {is_expected.to belong_to(:user)}
+    it {is_expected.to have_many(:transactions).dependent(:destroy)}
+  end
+  describe "Validations" do
+    subject{FactoryBot.build(:wallet)}
+    context "when field name" do
+      it {is_expected.to validate_presence_of(:name)}
+      it {is_expected.to validate_length_of(:name).is_at_most(Settings.digits.length_name_max_25)}
+    end
+    context "when field balance" do
+      it {is_expected.to validate_presence_of(:balance)}
+      it {is_expected.to validate_numericality_of(:balance).only_integer}
+    end
+  end
+  describe "scope" do
+    let!(:user) {FactoryBot.create :user}
+    let!(:wallet_1) {FactoryBot.create :wallet, user_id: user.id}
+    let!(:wallet_2) {FactoryBot.create :wallet, created_at: (Time.zone.now + 12), user_id: user.id}
+    let!(:wallet_3) {FactoryBot.create :wallet, created_at: (Time.zone.now + 55), user_id: user.id}
+    it "check scope wallet newest" do
+      expect(Wallet.newest).to eq([wallet_3, wallet_2, wallet_1])
+    end
+  end
+  describe "Public class methods" do
+    let!(:user) {FactoryBot.create :user}
+    let!(:wallet_1) {FactoryBot.create :wallet, balance: 3, user_id: user.id}
+    let!(:wallet_2) {FactoryBot.create :wallet, balance: 3, user_id: user.id}
+    let!(:wallet_3) {FactoryBot.create :wallet, balance: 3, user_id: user.id}
+    subject {Wallet}
+    describe "responds to its methods" do
+      it {is_expected.to respond_to :total_wallet}
+      it {is_expected.to respond_to :sum_balance}
+    end
+    describe "method .total_wallet" do
```

Comment:
```suggestion
    describe ".total_wallet" do
```
để lại tên các describe nhé
#instance_method
.class_method